tasks/libs/common/utils.py

```diff
@@ -70,6 +70,10 @@ def running_in_pyapp():
     return os.environ.get("PYAPP") == "1"
```

Comment:
Can you change this too ? I think it might crash in some cases
```suggestion
    return os.environ.get("PYAPP", 0) == "1"
```

Reply:
No, it would return `None` and would not crash. This is intended